test.sh

```diff
@@ -11,7 +11,7 @@ pytest -v tests/integration_tests/
 echo "\n Running e2e tests.  This may take a few minutes...\n"
 pytest -v tests/e2e_tests/
 
-# field tests are not run on the target machine where we can install
+# Field tests are typically run on the target machine where we can install
```

Comment:
TODO:  maybe just eliminate `tests/field_tests`

Field tests may be completely redundant with the scripts in `src/basic_bot/debug` which are self running (don't need to be run through pytest, but have to be run individually.  For example: 
```
python -m basic_bot.debug.test_opencv_capture
```